### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by johnwatson484**
# **What this PR does / why we need it**:
*A brief description of changes being made*
*Link to the relevant work items: e.g: Relates to ADO Work Item AB#213700 and builds on #3376 (link to ADO Build ID URL)*

# **Special notes for your reviewer**
*Any specific actions or notes on review?*

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [ ] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [ ] Title pattern should be `{work item number}: {title}`
- [ ] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests
- [ ] Version number in **package.json** has been updated

# **How does this PR make you feel**:
gif

### CODE REVIEW

File: app/routes/healthy.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
+const { GET } = require('../constants/http-verbs')
+const { OK } = require('../constants/ok')
+
 module.exports = {
-  method: 'GET',
+  method: GET,
   path: '/healthy',
```
Comment (simonnineteen90) [inline app/routes/healthy.js:6]: does this need to be prefixed with `/business`

File: app/routes/healthz.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
+const { GET } = require('../constants/http-verbs')
+const { OK } = require('../constants/ok')
+
 module.exports = {
-  method: 'GET',
+  method: GET,
   path: '/healthz',
```
Comment (simonnineteen90) [inline app/routes/healthz.js:6]: does this need to be prefixed with /business

File: helm/ffc-sfd-business/values.yaml [hunk 3]
```diff
@@ -14,12 +14,12 @@ deployment: {}
 # These may be changed. See here: https://github.com/DEFRA/adp-helm-library?tab=readme-ov-file#container-template
 container:
   memCpuTier: L
-  port: 3000
+  port: 3003
 
 # These may be changed. See here: https://github.com/DEFRA/adp-helm-library?tab=readme-ov-file#container-template
 livenessProbe:
   path: /healthz
```
Comment (simonnineteen90) [inline helm/ffc-sfd-business/values.yaml]: does this need to be prefixed with `/business`?

File: helm/ffc-sfd-business/values.yaml [hunk 4]
```diff
@@ -28,7 +28,7 @@ livenessProbe:
 # These may be changed. See here: https://github.com/DEFRA/adp-helm-library?tab=readme-ov-file#container-template
 readinessProbe:
   path: /healthy
```
Comment (simonnineteen90) [inline helm/ffc-sfd-business/values.yaml]: does this need to be prefixed with `/business`

File: scripts/start [hunk 5]
```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env sh
+if [ -z "$(docker network ls --filter name=^ffc-sfd$ --format={{.Name}})" ]; then
```
Comment (simonnineteen90) [inline scripts/start:2]: Think the network needs changing to `fcp-sfd`
On line 4 as well

File: docker-compose.link.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+networks:
+  default:
+    external:
+      name: ffc-sfd
```
Comment (simonnineteen90) [inline docker-compose.link.yaml:4]: Is this the correct network name? Saw `fcp-sfd` used on other repo

File: package.json
Comment (simonnineteen90) [inline package.json:1]: Version bump needed?

File: app/routes/healthz.js (hunk 2 above)
Comment (simonnineteen90) [reply app/routes/healthz.js:6]: not needed due to prefix in server setup

File: app/routes/healthy.js (hunk 1 above)
Comment (simonnineteen90) [reply app/routes/healthy.js:6]: not needed due to prefix in server setup

File: docker-compose.link.yaml (hunk 6 above)
Comment (simonnineteen90) [reply docker-compose.link.yaml:4]: not needed

Comment (johnwatson484) [reply docker-compose.link.yaml:4]: Should be ffc-sfd so can all run in local Docker network together

File: helm/ffc-sfd-business/values.yaml (hunk 3 above)
Comment (johnwatson484) [reply helm/ffc-sfd-business/values.yaml]: Yep

File: helm/ffc-sfd-business/values.yaml (hunk 4 above)
Comment (johnwatson484) [reply helm/ffc-sfd-business/values.yaml]: Yep

**Review by rtasalem — APPROVED** [review]
All looks good to me. Confirmed with Fay T. that there was nothing else to do for this PR. Only additional changes I made was updating one of the ports, the `appConfig`, and setting up the test for the `healthy` and `healthz` routes to pass.

**Pull request merged**